packfile.c

```diff
@@ -1646,6 +1646,12 @@ static void *read_object(struct repository *r,
 	return content;
 }
 
+static unsigned long g_nr_unpack_entry;
+unsigned long get_nr_unpack_entry(void)
```

Comment:
This needs to be static if it's not in the header file.

Reply:
Or marked `extern` in this file.

Reply:
got it.  thanks!  i was trying to not pollute the header file with an experimental function and then have to include it into all of the callers.